### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by sumitjoshi1989**
updated Dependencies
updated code to support latest javabrake
updated readme and changelog

### CODE REVIEW

File: src/main/java/io/airbrake/log4javabrake2/AirbrakeAppender.java [hunk 1]
```diff
@@ -124,7 +128,8 @@ static String formatLevel(Level level) {
   void send(Notice notice) {
     CompletableFuture.runAsync(() -> {
       if (this.notifier != null) {
-        this.notifier.sendSync(notice);
+       this.notifier.sendSync(notice);
+
```
Comment (mmcdaris) [inline src/main/java/io/airbrake/log4javabrake2/AirbrakeAppender.java]: This white space change looks accidental, probably a goodidea to revert for consistent tab use.

File: src/test/java/io/airbrake/log4javabrake2/TestAsyncSender.java [hunk 2]
```diff
@@ -1,19 +1,33 @@
...
+//     return new CompletableFuture<Notice>();
+//   }
+
+//   @Override
+//   public CompletableFuture<ApmResponse> send(String body, String path) {
+//     // TODO Auto-generated method stub
+//     return null;
+//   }
+
+// }
```
Comment (mmcdaris) [inline src/test/java/io/airbrake/log4javabrake2/TestAsyncSender.java]: Probably good to remove these and other comments from the file.

**Review by mmcdaris — APPROVED** [review]
LGTM, I left 2 non-blocking comments.

File: src/main/java/io/airbrake/log4javabrake2/AirbrakeAppender.java (hunk 1 above)
Comment (sumitjoshi1989) [reply src/main/java/io/airbrake/log4javabrake2/AirbrakeAppender.java]: Ok

File: src/test/java/io/airbrake/log4javabrake2/TestAsyncSender.java (hunk 2 above)
Comment (sumitjoshi1989) [reply src/test/java/io/airbrake/log4javabrake2/TestAsyncSender.java]: ok

**Review by abhishekpargal12 — APPROVED** [review]

**Pull request merged**